[2024-06-18T11:05:16.601+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-18T11:05:16.623+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_dag.thrid_task manual__2024-06-18T11:05:15.522643+00:00 [queued]>
[2024-06-18T11:05:16.633+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_dag.thrid_task manual__2024-06-18T11:05:15.522643+00:00 [queued]>
[2024-06-18T11:05:16.633+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-18T11:05:16.649+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): thrid_task> on 2024-06-18 11:05:15.522643+00:00
[2024-06-18T11:05:16.655+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3864) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-18T11:05:16.656+0000] {standard_task_runner.py:63} INFO - Started process 3877 to run task
[2024-06-18T11:05:16.656+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 'thrid_task', 'manual__2024-06-18T11:05:15.522643+00:00', '--job-id', '529', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmp4kh27r1y']
[2024-06-18T11:05:16.658+0000] {standard_task_runner.py:91} INFO - Job 529: Subtask thrid_task
[2024-06-18T11:05:16.703+0000] {task_command.py:426} INFO - Running <TaskInstance: test_dag.thrid_task manual__2024-06-18T11:05:15.522643+00:00 [running]> on host 195daacf7793
[2024-06-18T11:05:16.793+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_dag' AIRFLOW_CTX_TASK_ID='thrid_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T11:05:15.522643+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-18T11:05:15.522643+00:00'
[2024-06-18T11:05:16.795+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-18T11:05:16.813+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-18T11:05:16.814+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task3 and will be running after task1 at the same time as task2!']
[2024-06-18T11:05:16.826+0000] {subprocess.py:86} INFO - Output:
[2024-06-18T11:05:16.828+0000] {subprocess.py:93} INFO - hey, I am task3 and will be running after task1 at the same time as task2!
[2024-06-18T11:05:16.829+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-18T11:05:16.830+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-18T11:05:16.858+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=thrid_task, run_id=manual__2024-06-18T11:05:15.522643+00:00, execution_date=20240618T110515, start_date=20240618T110516, end_date=20240618T110516
[2024-06-18T11:05:16.911+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-18T11:05:16.929+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-18T11:05:16.932+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
